server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/cucumber/DownloadStepdefs.java

```diff
@@ -109,6 +110,17 @@ public void getDownloadWithWrongBlobId(String username) throws Throwable {
                 .get("/download/badbadbadbadbadbadbadbadbadbadbadbadbadb");
     }
 
+    @When("^\"([^\"]*)\" asks for a token for attachment \"([^\"]*)\"$")
+    public void postDownload(String username, String attachmentId) throws Throwable {
+        String blobId = blobIdByAttachmentId.get(attachmentId);
+        AccessToken accessToken = userStepdefs.tokenByUser.get(username);
```

Comment:
Sorry, I don't see the value of instantiating an Optional to do a ifPresent instead of a if != null.